app/views/content/non-uk-teachers/promos/_find-your-course.html.erb

```diff
@@ -6,7 +6,7 @@
   </div>
 
   <div class="right">
-    <span class="caption">Find your course</span>
+    <span class="caption-l caption--bold caption--purple">Find your course</span>
     <p>
       You can search for postgraduate courses that lead to QTS (not including
```

Comment:
```suggestion
      You can search for postgraduate courses that lead to qualified teacher status (QTS) (not including
```
@EllieNodder might as well update this one ahead of the acronym expander work - you may want to reword the Teach First bit now that there are two sets of brackets though?